elements/pfe-band/src/pfe-band.scss

```diff
@@ -170,66 +174,119 @@ $LOCAL: band;
+        --pfe-band--gridTemplateArea_mobile:
+          "body"
+          "aside"
+          "footer";
+        --pfe-band--gridTemplateArea_desktop:
+          "body aside"
+          "footer footer";
       }
       &[pfe-aside-mobile="top"] {
-        --pfe-band--gridTemplateArea_mobile: "aside" "body" "footer";
+        --pfe-band--gridTemplateArea_mobile:
+          "aside"
+          "body"
+          "footer";
       }
       &[pfe-aside-height="full"] {
-        --pfe-band--gridTemplateArea_desktop: "body aside" "footer aside" ;
+        --pfe-band--gridTemplateArea_desktop:
+          "body aside"
+          "footer aside" ;
       }
       &[pfe-aside-desktop="left"] {
-        --pfe-band--gridTemplateArea_desktop: "aside body" "footer footer";
+        --pfe-band--gridTemplateArea_desktop:
+          "aside body"
+          "footer footer";
         &[pfe-aside-height="full"] {
-          --pfe-band--gridTemplateArea_desktop: "aside body" "aside footer";
+          --pfe-band--gridTemplateArea_desktop:
+            "aside body"
+            "aside footer";
         }
       }
     }
     &[pfe-has-header][pfe-has-footer] {
-      --pfe-band--gridTemplateArea_mobile: "header" "body" "footer";
+      --pfe-band--gridTemplateArea_mobile:
+        "header"
+        "body"
+        "footer";
       &[pfe-has-aside] {
-        --pfe-band--gridTemplateArea_mobile: "header" "body" "footer" "aside";
-        --pfe-band--gridTemplateArea_desktop: "header header" "body aside" "footer footer";
+        --pfe-band--gridTemplateArea_mobile:
+          "header"
+          "body"
+          "footer"
+          "aside";
+        --pfe-band--gridTemplateArea_desktop:
+          "header header"
+          "body aside"
+          "footer footer";
       }
       &[pfe-aside-mobile="top"] {
-        --pfe-band--gridTemplateArea_mobile: "aside" "header" "body" "footer";
+        --pfe-band--gridTemplateArea_mobile:
+          "aside"
+          "header"
+          "body"
+          "footer";
       }
       &[pfe-aside-height="full"] {
-        --pfe-band--gridTemplateArea_desktop: "header aside" "body aside" "footer aside" ;
+        --pfe-band--gridTemplateArea_desktop:
+          "header aside"
+          "body aside"
+          "footer aside" ;
       }
       &[pfe-aside-desktop="left"] {
-        --pfe-band--gridTemplateArea_desktop: "header header" "aside body" "footer footer";
+        --pfe-band--gridTemplateArea_desktop:
+          "header header"
+          "aside body"
+          "footer footer";
         &[pfe-aside-height="full"] {
-          --pfe-band--gridTemplateArea_desktop: "aside header" "aside body" "aside footer";
+          --pfe-band--gridTemplateArea_desktop:
+            "aside header"
+            "aside body"
+            "aside footer";
         }
       }
     }
 
     position:  relative;
     margin:    0 auto;
-    width: #{pfe-local(Width)};
+    max-width: #{pfe-local(Width)};
```

Comment:
Doesn't max-width put the band at risk of being too short if the content doesn't fit?  And since it's centered we'd end up with inconsistently sized bands.  This needs to be width or min-width.

Reply:
Hmm, not sure what you mean? max-width != max-height
It does need `width:100%; max-width: #{pfe-local(Width)};` to ensure it fills out the available space though. good catch!

Reply:
updated b6b00181c23d195252f2b17fa0a5dd8a9f6ee9e9

Reply:
Short horizontally not vertically so still related to width.  The width: 100% fixes the issue though.  Just noting it needs both width and max-width or min-width.  +1